new/PRS13414Walatta_.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13414Walatta_" xml:lang="en" type="pers">
```

Comment:
An Id with _ is not ok: https://betamasaheft.eu/Guidelines/?id=entities-id-structure
Please, ask Pietro what would be the best way to correct it, if you can just do it on this branch changing the id and the name of the file.